sac/distributions/gmm.py

```diff
@@ -73,6 +75,9 @@ def _create_p_xz_params(self):
         log_sig_t = w_and_mu_and_logsig_t[..., 1+Dx:]
 
         log_sig_t = tf.minimum(log_sig_t, LOG_SIG_CAP_MAX)
+        log_sig_t = tf.maximum(log_sig_t, LOG_SIG_CAP_MIN)
```

Comment:
Just curious, did you actually see any benefit clipping `log_sig_t` below? I remember trying this as well when I was trying to catch NaNs, but in the end I did not observe these values never hitting the limit, so I left it out.